### ISSUE TITLE
Q-uploader upload() method only uploads the first of multiple files when called in an event handler. 

### ISSUE THREAD

**Issue by mjmare** [quasarframework/quasar#956]
I have a q-uploader field:
```
        <q-uploader
          ref="uploader"
          url="/api/upload"
          multiple
          @uploaded="uploaded"
          @add="added"
          :additionalFields="[{name: 'attachment_ref', value: this.form.attachment_ref}]"
        />
```
I want to upload the files as soon as they are selected, so I thought of calling the upload() method in the the event handler @add. 

```
      added (files) {
        console.log(files)
        this.$refs.uploader.upload()
      },
```

This almost works: the first file is uploaded automatically. All other files that were selected (simultaneously) are not uploaded, requiring a manual upload. 
In the example, the console log shows multiple files.

If I call the upload() method manually, all files *are* uploaded. 

Using quasar 0.14.4

**Comment by mjmare** [comment]
As a workaround, this works:

```
      added (files) {
        setTimeout(() => {
          this.$refs.uploader.upload()
        }, 200)
      },
```

So it seems like a timing issue.

**Comment by kevingermain** [comment]
I've experienced exactly the same problem and resolved with the same trick.

**Comment by rstoenescu** [comment]
This has been fixed in v0.15, so closing.
